src/common/configuration.ts

```diff
@@ -137,8 +141,5 @@ export function get() {
   return configuration;
 }
 
-if (!configuration) {
-  configuration = load();
-}
-
+configuration = load();
```

Comment:
Was also wondering about it first, but after thinking again it makes sense.
configuration defined and set to null some lines above on same file. So whenever this file executes (module loader imports it), configuration is part of that module. Means it is null when this line executes, always, because either booth lines are executed:
> let configuration: Configuration = null;
> if (!configuration) {
or none.

Now don't ask me if this code actually runs multiple times or not, I have lost track on how all those CommonJS, ES6, ESNext, .. module loaders work in detail after browser and node decided to go their own ways... but the general assumption always hold, if the global scope on that file executes, both lines execute, so configuration is null always.